server/src/main/java/com/cloud/server/ConfigurationServerImpl.java

```diff
@@ -926,16 +926,21 @@ private DiskOfferingVO createDefaultDiskOffering(String name, String description
         newDiskOffering.setUniqueName("Cloud.Com-" + name);
         // leaving the above reference to cloud.com in as it is an identifyer and has no real world relevance
         newDiskOffering.setSystemUse(isSystemUse);
-        newDiskOffering = _diskOfferingDao.persistDeafultDiskOffering(newDiskOffering);
+        newDiskOffering = _diskOfferingDao.persistDefaultDiskOffering(newDiskOffering);
         return newDiskOffering;
     }
 
     private ServiceOfferingVO createServiceOffering(long userId, String name, int cpu, int ramSize, int speed, String displayText,
             ProvisioningType provisioningType, boolean localStorageRequired, boolean offerHA, String tags) {
         tags = cleanupTags(tags);
+        DiskOfferingVO diskOfferingVO = new DiskOfferingVO(name, displayText, provisioningType, false, tags, false, false, false, true);
+        diskOfferingVO.setUniqueName("Cloud.Com-" + name);
```

Comment:
is this prefix "Cloud.Com-" required or offerings ?

Reply:
Here we are splitting the service offering and disk offering, so using the same name for both offerings.

Reply:
Post upgrade, the existing service offerings and related disk offerings have the same name. While creating a new service offering, this prefix may not be required. what do you think?

Reply:
Yes @rhtyd